sensing/gnss_poser/src/gnss_poser_core.cpp

```diff
@@ -89,8 +93,8 @@ void GNSSPoser::callbackNavSatFix(
 
   // calc gnss antenna orientation
   geometry_msgs::msg::Quaternion orientation;
-  if (use_ublox_receiver_ && nav_pvt_msg_ptr_ != nullptr) {
-    orientation = getQuaternionByHeading(nav_pvt_msg_ptr_->heading);
+  if (use_gnss_heading_ && gnss_heading_ != nullptr) {
+    orientation = getQuaternionByHeading(*gnss_heading_);
```

Comment:
Since `GnssInsOrientationStamped` already contains orientation, there is no need to execute this function.

Reply:
Fixed! https://github.com/autowarefoundation/autoware.universe/pull/1230/commits/4f58ddf026d6892b62d84085144e52c090d5e827